docs/sources/dashboards/previews.md

```diff
@@ -0,0 +1,112 @@
++++
+title = "Dashboard previews"
+keywords = ["grafana", "dashboard", "documentation", "previews"]
+aliases = ["/docs/grafana/latest/reference/previews/"]
+weight = 9
++++
+
+# Previews
+
+{{< figure  max-width="950px" src="/static/img/docs/dashboards/previews.jpg" animated-gif="/static/img/docs/dashboards/previews.gif" >}}
+
+> **Note:** Dashboard previews are available in Grafana 9.0+ as an opt-in beta feature. Please refer to the [permissions]({{< relref "#preview-visibility">}}) section to learn about issues with data source permissions before enabling the feature.
+
+Dashboard previews bring life to all dashboard list pages. Besides looking pretty, they can serve as an overview of all available dashboards, or as a tool to quickly find the right dashboard in case the dashboard names aren't enough on their own.
```

Comment:
```suggestion
Dashboard previews provide an overview of all available dashboards, or as a tool to quickly find the right dashboard in case the dashboard names aren't enough on their own.
```

Reply:
applied, split the sentence in two